pom.xml

```diff
@@ -72,6 +72,12 @@
 				<version>${uiframeworkVersion}</version>
 				<scope>provided</scope>
 			</dependency>
+			<dependency>
+				<groupId>org.openmrs.module</groupId>
```

Comment:
And what do we need this for?

Reply:
Without groupId how can I add a dependency to my module?

Reply:
I mean the entire dependency.

Reply:
I'm using this in the module  - see here(https://github.com/openmrs/openmrs-module-locationbasedaccess/pull/9/files#diff-0fa526dc98f8f95f9a0cceef8fdd699cR8)